src/libraries/System.Runtime.Numerics/tests/BigInteger/op_rightshift.cs

```diff
@@ -31,11 +31,46 @@ public static void LargeNegativeBigIntegerShiftTest()
             // Create a very large negative BigInteger
             BigInteger bigInt = new BigInteger(-1) << int.MaxValue;
             Assert.Equal(2147483647, bigInt.GetBitLength());
+            Assert.Equal(-1, bigInt.Sign);
 
-            // Right shift the BigInteger and ensure the operation behaves as expected
+            // Validate internal representation.
+            // At this point, bigInt should be a 1 followed by int.MaxValue zeros.
+            // Given this, bigInt._bits is expected to be structured as follows:
+            // - _bits.Length == (int.MaxValue + 1)/(8*sizeof(uint))
+            // - First (_bits.Length - 1) elements: 0x00000000
+            // - Last element: 0x80000000
+            //                   ^------ (There's the leading '1')
+
+            Assert.Equal(((uint) int.MaxValue + 1)/(8*sizeof(uint)), (uint) bigInt._bits.Length);
```

Comment:
Code styling is normally no space between the cast and its operand (so `(T)x`, not `(T) x; negatives get an extra set of parens, so `(T)(-x)`) and spaces between binary operators and their operands (so `x * y`, not `x*y`):
```suggestion
            Assert.Equal(((uint)int.MaxValue + 1) / (8 * sizeof(uint)), (uint)bigInt._bits.Length);
```

Would be good to fix up the two comments as well